blockbase/theme.json

```diff
@@ -173,8 +173,9 @@
 				}
 			],
 			"fontSizes": {
-				"x-small": "14px",
-				"normal": "18px"
```

Comment:
`custom-normal` which is used as default font, is more than `preset-normal` and it sets different fonts when patterns inserted. Is it a good idea to use the preset-normal with 18px instead?

Reply:
I think we only want to keep the --wp--preset--font-size's to only the four Gutenberg provides (small, medium, large, x-large), due to the problems when a user sets content to a font size that might be undefined in other themes.  Better to use an explicit size in those circumstances (such as the font-size for the H1).

The sizes look good, I notice no changes in the rendered page.

Reply:
+1 to @pbking 's feedback. I think we want to drop `normal` and `huge` from the `fontSizes` array. That way the font size picker will look like this:

<img width="267" alt="Screen Shot 2022-03-25 at 11 06 21 AM" src="https://user-images.githubusercontent.com/5375500/160147331-35f1f4fc-58a8-402d-bf4f-9dfc2f786240.png">

On this PR it looks like this:

<img width="267" alt="Screen Shot 2022-03-25 at 10 57 20 AM" src="https://user-images.githubusercontent.com/5375500/160147372-602a64b8-eb3e-4025-ab20-7d370ce8a842.png">

Reply:
Agreed. Removed normal and huge fonts from preset sizes.